src/gei/Program.cs

```diff
@@ -21,10 +21,14 @@ public static async Task Main(string[] args)
                 .AddCommands()
                 .AddSingleton(Logger)
                 .AddSingleton<EnvironmentVariableProvider>()
+                .AddSingleton<BlobServiceClientFactory>()
+                .AddTransient<IBlobServiceClientFactory>(sp => sp.GetRequiredService<BlobServiceClientFactory>())
                 .AddSingleton<GithubApiFactory>()
+                .AddSingleton<AzureApiFactory>()
                 .AddSingleton<AdoApiFactory>()
                 .AddTransient<ITargetGithubApiFactory>(sp => sp.GetRequiredService<GithubApiFactory>())
                 .AddTransient<ISourceGithubApiFactory>(sp => sp.GetRequiredService<GithubApiFactory>())
+                .AddTransient<IAzureApiFactory>(sp => sp.GetRequiredService<AzureApiFactory>())
```

Comment:
Same as above:
```c#
AddTransient<IAzureApiFactory, AzureApiFactory>()
```

Reply:
Updated in https://github.com/github/gh-gei/pull/221/commits/20124f53f834ac969c213e17bdb8caa505a3fcee